compiler-core/src/language_server/engine.rs

```diff
@@ -225,6 +230,8 @@ where
                 }
 
                 Located::Arg(_) => None,
+
+                Located::Annotation(_, _) => Some(this.completion_types(module)),
```

Comment:
Let's have some tests for this new functionality or remove it.

Reply:
Yup sounds good :D for some reason I thought this was covered by existing tests